.github/CODEOWNERS

```diff
@@ -1,3 +1,3 @@
 * @cdr/code-server-reviewers
 
-ci/helm-chart @Matthew-Beckett @alexgorbatchev
+ci/helm-chart/ @Matthew-Beckett @alexgorbatchev
```

Comment:
The Missing Slash [Slinging Slasher]!

![image](https://user-images.githubusercontent.com/3806031/134998588-55e24f91-50cb-499b-b24f-e465b01e7647.png)